services/ui_backend_service/tests/integration_tests/metadata_test.py

```diff
@@ -138,18 +138,18 @@ async def test_list_metadata_attempt_id_filter(cli, db):
     _metadata_second["attempt_id"] = 1
 
     # no filter
-    await _test_list_resources(cli, db, "/flows/{flow_id}/runs/{run_number}/metadata?order=ts_epoch".format(**_task), 200, [_metadata_second, _metadata_first, _metadata_global])
+    await _test_list_resources(cli, db, "/flows/{flow_id}/runs/{run_number}/metadata?_order=ts_epoch".format(**_task), 200, [_metadata_second, _metadata_first, _metadata_global])
```

Comment:
`order` => `_order`.  It never worked because we only look for `_order` on backend...